ckan/requirements.txt

```diff
@@ -14,11 +14,11 @@ ckanext-dcat @ git+https://github.com/ckan/ckanext-dcat@c4e7319aff55e5315831b28e
 ckanext-envvars @ git+https://github.com/GSA/ckanext-envvars.git@33f7e190ab332244cb961a425e09af592d9b647b
 -e git+https://github.com/GSA/ckanext-geodatagov.git@bc700f671cd1d890db265c65f5db1cba121297f4#egg=ckanext_geodatagov
 ckanext-googleanalyticsbasic @ git+https://github.com/GSA/ckanext-googleanalyticsbasic.git@c6a425d5e14d658c0fa3661fdc4423162161c3f4
--e git+https://github.com/GSA/ckanext-harvest.git@fd5a6b739666a62cff486873877eef10523aec1a#egg=ckanext_harvest
+-e git+https://github.com/GSA/ckanext-harvest.git@b653113abc059f58917732f33a3ae2c84a476746#egg=ckanext_harvest
 -e git+https://github.com/gsa/ckanext-spatial.git@36981c39d624685bc85b84f801f2a2d6eaea67f6#egg=ckanext_spatial
 ckantoolkit==0.0.3
 click==7.1.2
-cryptography==3.4.8
+cryptography==35.0.0
```

Comment:
```suggestion
cryptography==3.5.0
```

Reply:
Oh, I saw the link now.. that's a pretty big mess up haha 😓

Reply:
Until it's registered on pypi, we have to use the published version... https://pypi.org/project/cryptography/#history